sql/core/src/test/scala/org/apache/spark/sql/JoinHintSuite.scala

```diff
@@ -507,8 +507,6 @@ class JoinHintSuite extends PlanTest with SharedSparkSession with AdaptiveSparkP
           BuildLeft)
 
         // Shuffle-hash hint specified but not doable
-        assertBroadcastHashJoin(
-          sql(equiJoinQueryWithHint("SHUFFLE_HASH(t1)" :: Nil, "left")), BuildRight)
```

Comment:
Shall we still keep this `shuffle-hash` hint test but change the assert to `assertShuffleHashJoin`?